packages/e2e-tests/test/e2e-oidc.spec.ts

```diff
@@ -413,8 +413,15 @@ describe('OIDC auth e2e', function () {
     // We cannot make the mongod server accept the mock IdP's certificate,
     // so the best we can verify here is that auth failed *on the server*
     shell.assertContainsOutput(/MongoServerError: Authentication failed/);
+  });
```

Comment:
The test here should probably be renamed, and `--tlsUseSystemCA` dropped from its command line, right?

Reply:
Ah actually, the test should be definately renamed, but I wanted to keep `--tlsUseSystemCA` so we verify that setting this cli option does not error. The test right under this does not set `--tlsUseSystemCA`.

Reply:
@alenakhineika I think the test here still needs to be renamed, right?